python/revng/internal/cli/_commands/schema_diff.py

```diff
@@ -0,0 +1,86 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import yaml
+from revng.internal.cli.commands_registry import Command, CommandsRegistry
+import copy
```

Comment:
Make sure you run `revng check-conventions` before pushing. The `copy` import is unused.

Reply:
Done.